scripts/fix_diffs.py

```diff
@@ -94,11 +94,17 @@ def main(fp):
 
 if __name__ == '__main__':
     import sys
+
     if len(sys.argv) < 2:
-        print('Usage: python fix_diffs.py <po_file_path>')
+        print('Usage: python fix_diffs.py <po_file_path> <original_po_file_commit_hash>')
 
-    fp = sys.argv[1]
-    output_lines = main(fp)
+    fp = './' + sys.argv[1]
```

Comment:
I'm curious about this line, can you please explain the reason behind this line of code? As far as I understand, a relative path defaults to be relative to the current directory. What is an example case you're trying to resolve with this?

Reply:
I found an error when I was running the script in the subdirectory 'tutorial' (or any subdirectory like '/reference',' /howto'...etc):
(Actually, I don't know how to explain this bug in detail 😅. Maybe the image below could reveal this bug.)
<img width="785" alt="2018-06-10 1 53 37" src="https://user-images.githubusercontent.com/18432820/41198684-0c4d1cfa-6cb6-11e8-884f-14794b9656ee.png">

Reply:
I see where the problem is, this is because I was passing the filename argument directly to `git show`, however the git object indicating the file at HEAD should be `HEAD:<path>` where the path argument is by default relative to the root of the git repo.

Thanks! ✨